main.py

```diff
@@ -1,7 +1,63 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(
+        name="Western",
+    )
+    Genre.objects.create(
+        name="Action",
+    )
+    Genre.objects.create(
+        name="Dramma",
```

Comment:
Typographical Error: Correct the genre name from "Dramma" to "Drama" initially to avoid unnecessary updates later.